docs/clusters/alpine/allocations.md

```diff
@@ -43,23 +43,25 @@ resources requested, job size, and QOS.
 ### Allocations
 When you receive a Research Computing account you are automatically 
 assigned a **Trailhead Auto-Allocation**, which grants you a fixed share 
-of `ucb-general`. The Trailhead is a great allocation for smaller jobs or 
+of `ucb-general`. RMACC users are granted a fixed share of `rmacc-general`. 
+The Trailhead is a great allocation for smaller jobs or 
 testing and benchmarking your code. However, if you consume more than your 
-fixed share of `ucb-general`,  your `LevelFS` will decrease, causing your 
```

Comment:
remove the word 'fixed' from this sentence, since the word is used frequently in these sentences. Rather, "However, if you consume more than your allocated share of `ucb-general`..."